apps/server/swagger/schemas.yml

```diff
@@ -9,8 +9,8 @@ components:
           schema:
             $ref: '#/components/responses/Error'
 
-    StateConflict:
-      description: State Conflict
+    StatusConflict:
+      description: Status Conflict
```

Comment:
This description is just the name. It doesn't tell me what a StatusConflict represents, what can cause it, what information I might get from the error, etc.